pages.fr/common/ansiweather.md

```diff
@@ -0,0 +1,16 @@
+# ansiweather
+
+> Un script Shell pour afficher les conditions météo actuelles dans votre terminal.
+> Plus d'informations : <https://github.com/fcambus/ansiweather>.
+
+- Affiche une prévision avec le système métrique pour les 5 prochains jours dans la ville de Paris, France :
+
+`ansiweather -u {{metric}} -f {{5}} -l {{Paris,FR}}`
+
+- Affiche une prévision avec des symboles et les données d'ensoleillement sur votre position actuelle :
+
+`ansiweather -s {{true}} -d {{true}}`
+
+- Affiche une prévision avec les données sur le vent et l'humidité sur votre position actuelle :
```

Comment:
I suggest preventing using several times `sur` in the same sentence :

```suggestion
- Affiche une prévision avec des symboles et les données d'ensoleillement pour votre position actuelle :

`ansiweather -s {{true}} -d {{true}}`

- Affiche une prévision avec les données sur le vent et l'humidité pour votre position actuelle :
```

Reply:
I'm not sure about `pour`. Most of the time I use/hear `sur` about the localisation. 

Like in the army when they say `Restez sur vos positions`

But `pour` works in that case, i'll change it :D